templates/polling_location/polling_location_visualize.html

```diff
@@ -4,7 +4,7 @@
 
 {% block js %}
   <script src="{% static "js/PollingLocationVisualization.js" %}" type="text/javascript"></script>
-  <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyCM0vM1mzh56OuA5IfgW_U6Qd9jD99WH6M&callback=initMap" async defer></script>
```

Comment:
This should use GOOGLE_MAPS_API_KEY from environment)variables.json, instead of hard coding the key value, even if it is not secret.  That way it can be updated in one place when necessary.  If it is secret the value should be rolled.